<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>charsWindow</class>
 <widget class="QMainWindow" name="charsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#charsWindow {
	background-color: rgb(0, 0, 90);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="charsFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>400</width>
      <height>230</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#charsFrame{
	background-color: rgb(50, 50, 255)
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="openChar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#openChar{
	background-color: rgb(68, 68, 68);
	border: 1px ;
	border-radius: 20px
}

#openChar:hover{
	background-color: rgb(42, 42, 42);
	border: 1px ;
	border-radius: 20px;
}</string>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createChar">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#createChar{
	background-color: rgb(68, 68, 68);
	border: 1px ;
	border-radius: 20px
}

#createChar:hover{
	background-color: rgb(42, 42, 42);
	border: 1px ;
	border-radius: 20px;
}</string>
     </property>
     <property name="text">
      <string>New</string>
     </property>
    </widget>
    <widget class="QListWidget" name="charList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>380</width>
       <height>140</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Fixedsys</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#charList{
	background-color: rgb(186, 186, 186);
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refreshButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>170</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#refreshButton{
	background-color: rgb(68, 68, 68);
	border: 1px ;
	border-radius: 20px
}

#refreshButton:hover{
	background-color: rgb(42, 42, 42);
	border: 1px ;
	border-radius: 20px;
}</string>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
